src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -412,7 +412,7 @@ export function MultiSelect<T>({
             <IconButton
                 disabled={disabled}
                 name="operation-clear"
-                data-test-id="clear-all-items"
+                data-testid="clear-all-items"
```

Comment:
Please keep `data-test-id`. Our lib used it like that long before, just changing it now would lead to incompatibility, e.g. with currently existing e2e tests, ect, pp. We already started to support `data-testid` and also this test id for the icon button is already marked as deprecated. We change this later when we do a next major release: https://github.com/eccenca/gui-elements/pull/155/commits/08338e65a1b43fe06ceb2a83a2602c6b2353c668#diff-a5d5d7f7576466bc7d1f3674cb021bb2020e85282f2f42043b4609390227bcdfR420